chapters/hi/_toctree.yml

```diff
@@ -27,7 +40,6 @@
     title: डेटा संसाधित करना
   - local: chapter3/3
     title: मॉडल कि फाइन-ट्यूनिंग Trainer API या Keras के साथ
-    local_fw: { pt: chapter3/3, tf: chapter3/3_tf }

```

Comment:
I think we need to revert this change in order to build the docs. Is there a reason why you needed to remove it?

Reply:
The changes are reversed. Have a look and let me know. @lewtun

Reply:
Thanks, this looks great!